packages/core/contracts/financial-templates/funding-rate-store/implementation/FundingRateStore.sol

```diff
@@ -1,15 +1,41 @@
 pragma solidity ^0.6.0;
 pragma experimental ABIEncoderV2;
 
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/SafeERC20.sol";
+
+import "../../../oracle/interfaces/StoreInterface.sol";
+import "../../../oracle/interfaces/OracleInterface.sol";
+import "../../../oracle/interfaces/FinderInterface.sol";
+import "../../../oracle/implementation/Constants.sol";
+
 import "../interfaces/FundingRateStoreInterface.sol";
 import "../../../common/implementation/Testable.sol";
+import "../../../common/implementation/FixedPoint.sol";
 
 
 contract FundingRateStore is FundingRateStoreInterface, Testable {
+    using SafeMath for int256;
```

Comment:
So just remove this you think?

Reply:
I'm going to correct this to `SafeMath for uint256` because we will need SafeMath when doing math on determining the last publish time. More on that in another comment

Reply:
SGTM